sf_stats.Api/sf_stats.DataAccess.MSSql/Context/SFStatDbContext.cs

```diff
@@ -19,7 +19,12 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
             modelBuilder.HasDefaultSchema("dbo");
 
-            modelBuilder.ApplyConfiguration(new LogEntityConfiguration());
+            modelBuilder.ApplyConfiguration(new LogEntityConfiguration())
+                        .ApplyConfiguration(new SeasonEntityConfiguration())
+                        .ApplyConfiguration(new DivisionEntityConfiguration())
+                        .ApplyConfiguration(new TeamEntityConfiguration())
+                        .ApplyConfiguration(new StatTypeEntityConfiguration())
+                        .ApplyConfiguration(new PlayerEntityConfiguration());
```

Comment:
I'm not sure if this will work chaining like that but I don't know. Were you able to generate the tables properly like that. I thought it needed to look like the suggestion below

```suggestion
            modelBuilder.ApplyConfiguration(new LogEntityConfiguration());
            modelBuilder.ApplyConfiguration(new SeasonEntityConfiguration());
            modelBuilder.ApplyConfiguration(new DivisionEntityConfiguration());
            modelBuilder.ApplyConfiguration(new TeamEntityConfiguration());
            modelBuilder.ApplyConfiguration(new StatTypeEntityConfiguration());
            modelBuilder.ApplyConfiguration(new PlayerEntityConfiguration());
```

Reply:
Looks like `.ApplyConfiguration` returns the same `ModelBuilder` instance specifically for chaining:
[ModelBuilder.ApplyConfiguration Method](https://docs.microsoft.com/en-us/dotnet/api/microsoft.entityframeworkcore.modelbuilder.applyconfiguration?view=efcore-2.0)